exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -192,8 +192,8 @@ const TEN_POINT_SCALE = [
     'number9',
     'number10'
 ];
-const SEVEN_POINT_SCALE = TEN_POINT_SCALE.slice(0, 8);
-const NINE_POINT_SCALE = TEN_POINT_SCALE.slice(0, 10);
+const SEVEN_POINT_SCALE = TEN_POINT_SCALE.slice(1, 8);
```

Comment:
I accidentally included one extra item (instead of 0-7, the scale is supposed to go from 1-7)